bundle/mvn/default-bundle/src/main/resources/application.properties

```diff
@@ -1,5 +1,15 @@
-camunda.connector.polling.enabled=true
-camunda.connector.polling.interval=5000
+#camunda.connector.polling.enabled=true
+#camunda.connector.polling.interval=5000
 
-camunda.connector.webhook.enabled=true
-#spring.main.web-application-type=none
\ No newline at end of file
+#camunda.connector.webhook.enabled=true
+##spring.main.web-application-type=none
+
+# TODO : revisit this
+# Test properties for localhost
+server.port=9898
+zeebe.broker.gateway-address=localhost:26500
+zeebe.client.security.plaintext=true
+
+management.server.port=9080
+management.context-path=/actuator
+management.endpoints.web.exposure.include=metrics,health,prometheus
```

Comment:
Do we need changes in this file?

Reply:
Good catch, we do not need these changes, it was necessary for local testing only. Will remove these changes.